code/modules/mob/living/status_procs.dm

```diff
@@ -392,12 +392,20 @@
 		S.duration = amount + world.time
 	else if(amount > 0)
 		S = apply_status_effect(STATUS_EFFECT_SLEEPING, amount)
```

Comment:
Might want to include `voluntary` in the parameters of the `/datum/status_effect/incapacitating/sleeping/on_creation` override

Reply:
Done.
Apologies, must have somehow missed this comment last time.